frontend/src/scenes/short-links/ShortLinkNewScene.tsx

```diff
@@ -0,0 +1,144 @@
+                                        className="flex-1"
+                                        autoWidth={false}
+                                    />
+                                </LemonField>
+                            </div>
+                        </div>
+                        
+                        <LemonField name="tags" label="Tags">
+                                <LemonInputSelect
+                                    placeholder="Select tags..."
+                                    mode="multiple"
+                                    allowCustomValues
+                                    fullWidth
+                                    autoWidth={false}
+                                />
+                        </LemonField>
+
+                        <LemonField name="description" label="Comments">
+                            <LemonTextArea
+                                placeholder="Add comments"
+                                minRows={2}
+                            />
+                        </LemonField>
+                    </div>
+
+                    <LemonDivider vertical />
+                    
+                    <div className="flex-1 space-y-6 max-w-80">
+                        <div>
+                            <div className="flex justify-between items-center">
+                                <LemonLabel>
+                                    <span className="flex items-center gap-1">
+                                        QR Code
+                                    </span>
+                                </LemonLabel>
+                                <div className="flex flex-row">
+                                    <LemonButton
+                                        icon={<IconDownload />}
+                                        size="xsmall"
+                                        onClick={() => {}}
+                                        tooltip="Download QR code"
+                                    />
+                                    <LemonButton
+                                        icon={<IconCopy />}
+                                        size="xsmall"
+                                        onClick={() => {}}
+                                        tooltip="Copy to clipboard"
+                                    />
+                                </div>
+                            </div>
+                            
+                            <div className="border rounded-md p-4 mt-2 bg-bg-light flex items-center justify-center">
+                                <div className="text-center">
+                                    <QRCodeSVG 
+                                        size={128} 
+                                        value="https://reactjs.org/"
+                                        imageSettings={{
+                                            src: '/static/posthog-icon.svg',
```

Comment:
this is so pretty, great job :)

We should let people customize this:
- Teams plan can add their own/remove our logo

Reply:
this will also be a followup pr